include/benchmark/benchmark_api.h

```diff
@@ -414,16 +416,18 @@ class State {
   // Range arguments for this run. CHECKs if the argument has been set.
   BENCHMARK_ALWAYS_INLINE
   int range_x() const {
```

Comment:
we should mark these as deprecated as they're largely useless with your change. at some point  we should remove them.